projects/igniteui-angular/src/lib/navigation-drawer/navigation-drawer.component.spec.ts

```diff
@@ -622,12 +622,26 @@ class TestComponentDIComponent {
     public drawerWidth: string | number;
 }
 
-class TestComponentPin extends TestComponentDIComponent {
+@Component({
+    selector: 'igx-test-cmp-pin',
+    providers: [IgxNavigationService],
+    template: '<igx-nav-drawer></igx-nav-drawer>',
+    standalone: true,
+    imports: [IgxNavigationDrawerComponent, IgxNavDrawerTemplateDirective, IgxNavDrawerMiniTemplateDirective, NgIf]
+})
+class TestComponentPinComponent extends TestComponentDIComponent {
```

Comment:
Could've been `TestPinComponent` to follow naming convention and avoid duplicating the name, but truly not important anyway.